src/app/basicauction/components/AuctionDropDown.css.ts

```diff
@@ -0,0 +1,54 @@
+import { style } from '@vanilla-extract/css';
+
+import colors from '@/styles/color';
+import typography from '@/styles/typo';
+
+const borderColor = `1px solid ${colors.gray6}`;
+const dropdownWidth = '130px';
+const dropdownSpacing = '8px';
+
+export const dropdownContainer = style({
+  display: 'flex',
+  justifyContent: 'flex-end',
+  position: 'relative',
+});
+
+export const dropdown = style({
+  ...typography.caption,
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  gap: '10px',
+  width: dropdownWidth,
+  padding: dropdownSpacing,
+  border: borderColor,
+  boxSizing: 'border-box',
+  cursor: 'pointer',
+});
+
+export const dropdownContent = style({
+  position: 'absolute',
+  top: '100%',
+  display: 'flex',
+  flexDirection: 'column',
+  border: borderColor,
+  boxSizing: 'border-box',
+  background: 'white',
+  zIndex: 1,
```

Comment:
```suggestion
  backgroundColor: 'rgba(255, 255, 255, 0.8)',
  zIndex: 10,
```
이렇게 되어야 dim 위로 dropdown이 올라올 수 있네요~